### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Samsonroyal**
I created a new file, called CONTRIBUTING.md; and added all the necessary steps a developer can follow when wanting to contribute to this project. Please review the changes and merge the PR if it meets criteria.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+We welcome contributions from everyone, regardless of their level of experience or technical background. However, we ask that you follow the guidelines as clearly shown in the README.md and below to help us when reviewing your Pull Request in order to integrate your contributions quickly and smoothly. Read <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request">here</a> on how to make a Pull Request.
+
+## Getting Started
+
+To get started, you should:
+
+1. Fork the Autotelegram repository on GitHub
+2. <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop">Clone your fork</a> to your local machine.
+3. Install the development dependencies using ` pip install -r requirements-dev.txt `
```
Comment (HassanBahati) [inline CONTRIBUTING.md:39]:
```suggestion
3. Install the development dependencies using ` pip install -r requirements.txt `

```

i think its [requirements.txt](https://github.com/OSCA-Kampala-Chapter/autotelegram/blob/main/requirements.txt) not `requirements-dev.txt`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+- Improve existing code
+- Add new functionality
+
+We welcome contributions from everyone, regardless of their level of experience or technical background. However, we ask that you follow the guidelines as clearly shown in the README.md and below to help us when reviewing your Pull Request in order to integrate your contributions quickly and smoothly. Read <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request">here</a> on how to make a Pull Request.
+
+## Getting Started
+
+To get started, you should:
+
+1. Fork the Autotelegram repository on GitHub
```
Comment (HassanBahati) [inline CONTRIBUTING.md:37]:
```suggestion
1. Fork the [Autotelegram repository](https://github.com/OSCA-Kampala-Chapter/autotelegram) on GitHub

```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+2. <a href="https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop">Clone your fork</a> to your local machine.
+3. Install the development dependencies using ` pip install -r requirements-dev.txt `
+4. Read the <a href="https://osca-kampala-chapter.github.io/autotelegram/">documentation</a> to learn how to use the tool and how to run tests.
```
Comment (HassanBahati) [inline CONTRIBUTING.md:40]: This doesn't seem to really get to address the `how to use the tool and how to write tests`.....as it the link leads to `documentation home page`....what that link lead to [TESTS.md](https://github.com/OSCA-Kampala-Chapter/autotelegram/pull/57/files#diff-6129e4842294d2f7cbef7f74cda4f4457150fa9c642210659bffbdb51c092994) which this PR adds?

**Review by HassanBahati — CHANGES_REQUESTED** [review]

**Review by HassanBahati — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (Samsonroyal) [reply CONTRIBUTING.md:40]: The documentation at the official autotelegram site is still under development and ideally, a user should be able to access all the necessary documentation from there once we've fully developed it. The TESTS.md has a very narrow scope

**Comment by ht-thomas** [comment]
@Samsonroyal could you copy the content from CONTRIBUTING.md into docs/contributing.md instead. This is because that file is already linked to be automatically built with the docs. Also place the TESTS.md file in the docs folder,

**Comment by Samsonroyal** [comment]
@ht-thomas let me do that then push new commits

**Comment by Samsonroyal** [comment]
I am going to close this PR and open a new one where I have applied all the changes

**Pull request closed without merging**